lib/puppet.rb

```diff
@@ -80,7 +79,7 @@ class << self
         else
           GettextSetup.initialize(locale_path, :file_format => :mo)
         end
-        FastGettext.locale = GettextSetup.negotiate_locale(Locale.current.language)
+        GettextSetup.negotiate_locale!(GettextSetup.candidate_locales)
```

Comment:
I assume `FastGettext` automatically picks up on `Gettextsetup` configuration?

I don't know how the various libs plug together.